Automattic/wp-calypso
# Desktop: simplify external links handler

### Description

Previously we were looking at the URL which a link was going, and in many circumstances opening it in an external browser window instead of the app.

With the new navigation in the title bar it makes sense to allow going to all links inside the app as you can still get back to WordPress with a single click.

### Testing

- Run the app from the desktop folder with `yarn run dev`
- Click through the reader
- Click on different links and ensure you stay in the app
- Should not get stuck anywhere, should always be able to click the back, forward, or WordPress icon to get back
